cfn/README-GD.md

```diff
@@ -1,63 +1,64 @@
 # guardduty.template
 
-Alert Logic Amazon Web Services (AWS) CloudWatch Events (CWE) Collector CloudFormation template.
+Alert Logic Amazon Web Services (AWS) CloudWatch Events (CWE) Collector CloudFormation template (CFT).
 
 # Overview
 
-This folder contains the AWS CWE JavaScript lambda function and the CloudFormation template (CFT) that deploys the GuardDuty events collector to AWS. The GuardDuty collector collects and forwards CloudWatch events to the Cloud Insight backend for display as threats on the Incidents page.
+This folder contains the AWS CWE JavaScript lambda function and the AWS CloudFormation template that deploys the GuardDuty events collector to AWS. The GuardDuty collector collects and forwards CloudWatch events to the Cloud Insight Essentials backend for display as threats on the Incidents page.
 
-Amazon GuardDuty is a continuous security monitoring service that requires no customer-managed hardware or software. 
-GuardDuty analyzes and processes VPC Flow Logs and AWS CloudTrail event logs. GuardDuty uses security logic and 
-AWS usage statistics techniques to identify unexpected and potentially unauthorized and malicious activity. 
-Such activity includes escalations of privileges, uses of exposed credentials, or communication with 
-malicious IPs, URLs, or domains. GuardDuty informs you of the status of your AWS infrastructure and applications by producing security `findings`. 
+Amazon GuardDuty is a continuous security monitoring service that requires no customer-managed hardware or software. GuardDuty analyzes and processes VPC Flow Logs and AWS CloudTrail event logs. GuardDuty uses security logic and AWS usage statistics techniques to identify unexpected and potentially unauthorized and malicious activity, like escalations of privileges, uses of exposed credentials, or communication with malicious IPs, URLs, or domains.
 
 # Before you begin
 
-This procedure requires administrative permissions in AWS and your Alert Logic Cloud Insight account.  You also need to download the Alert Logic custom CFT to your local machine from [the Alert Logic public github repository](https://github.com/alertlogic/cwe-collector/blob/master/cfn/guardduty.template).
+To perform this procedure you must have administrative permissions in your AWS and Alert Logic Cloud Insight Essentials accounts. You must also download the Alert Logic custom CFT to your local machine from [the Alert Logic public github repository](https://github.com/alertlogic/cwe-collector/blob/master/cfn/guardduty.template).
```

Comment:
It is not required to download the template.
The template URL `https://s3.amazonaws.com/alertlogic-collectors-us-east-1/cfn/guardduty.template` can be used.

Reply:
We need to point the customer to a single download location to avoid confusion. Our public github location is the URL used within the product. For example, the GuardDuty remediation text points customers to our github to download the template (unless that was changed, and I am not aware of it). The documentation should be consistent with content in the product. I'd like to leave this as-is until we determine which central location we want customers to use as a download URL.

Reply:
The point is there is no need to download the template in order to run it.
Usage of template URL is faster and easier.
The readme doesn't have any futher references to downloaded file 
and  `### Use the Amazon console to deploy` section describes URL way of CFT deployment.

Reply:
Good point on the AWS console deployment section. In which case, we should probably talk to the UI team about updating the content of the GuardDuty remediation, which likely still refers to the github link.